README.md

```diff
@@ -24,10 +24,27 @@ def application do
 end
 
 def deps do
-  [{:slack, "~> 0.19.0"}]
+  [{:slack, "~> 1.0.0"}]
```

Comment:
This change is implying that this will bring the library to a stable `1.0.0` which I don't think is the case. Can we remove the version number bumps so I can do that in another PR?

Reply:
How should we bump to signify this is breaking change though? Based on semantic versioning, a major release is one that isn't compatible with a prior release. This is a big enough change that it isn't compatible with the existing system.

Reply:
Usually pre-1.0 releases aren't considered to have a stable API so it's generally okay to make breaking changes before we hit 1.0. Semver does bring that up [here](https://semver.org/#spec-item-4).

Going to 1.0 is a commitment, so I just want to make sure the library is okay with sticking to that API for a while. My big hesitance for not going 1.0 is the web API, since it needs some attention.

Reply:
Can you update this to `0.20.0` as well? Once that's updated, happy to merge this. :)

Reply:
@BlakeWilliams all set!